src/main/java/io/vertx/core/eventbus/impl/HandlerRegistration.java

```diff
@@ -58,13 +58,13 @@ void receive(MessageImpl msg) {
 
   protected abstract void dispatch(Message<T> msg, ContextInternal context, Handler<Message<T>> handler);
 
-  synchronized void register(String repliedAddress, boolean localOnly, Promise<Void> promise) {
+  synchronized void register(boolean broadcast, boolean localOnly, Promise<Void> promise) {
     if (registered != null) {
       throw new IllegalStateException();
     }
-    registered = bus.addRegistration(address, this, repliedAddress != null, localOnly, promise);
+    registered = bus.addRegistration(address, this, broadcast, localOnly, promise);
     if (bus.metrics != null) {
-      metric = bus.metrics.handlerRegistered(address, repliedAddress);
+      metric = bus.metrics.handlerRegistered(address, null /* regression */);
```

Comment:
Have you got any ideas already about how to address this regression?

Perhaps can we use a constant for now as a signal for a reply handler?

For the long term we shall pass the `broadcast` param to the SPI implementation.

FYI, Micrometer SPI impl does not use this value.

Reply:
I think we should simply remove it from the metrics as it seems to be limiting the protocol.

Reply:
I need to check dropwizard and if that's so we simply remove it

Reply:
it is actually not used in dropwizard nor micrometer, so it can be safely removed